### DISCUSSION TITLE
Setting up `*.js` files containing `gql` template strings

### DISCUSSION THREAD

**Discussion by dharkness** [graphql-hive/graphql-eslint#986] [Q&A] [answered]
The configuration seems pretty straightforward for this, but I'm getting errors when running `yarn lint`. I do not have any `*.schema` files and instead, use `makeExecutableSchema` with multiple `typeDefs.js` files.

Here's my `.eslintrc`:

```json
{
  "parser": "@babel/eslint-parser",
  "plugins": ["class-property", "import"],
  "parserOptions": {
    "ecmaVersion": 11,
    "sourceType": "module",
    "ecmaFeatures": {
      "experimentalObjectRestSpread": true
    },
    "operations": "./src/**/*.js"
  },
  "overrides": [
    {
      "files": ["*.js"],
      "processor": "@graphql-eslint/graphql"
    },
    {
      "files": ["*.graphql"],
      "extends": "plugin:@graphql-eslint/schema-recommended"
    }
  ],
  ...
}
```

I also tried without the recommended rule set as shown in the README:

```json
    {
      "files": ["*.graphql"],
      "parser": "@graphql-eslint/eslint-plugin",
      "plugins": ["@graphql-eslint"],
      "rules": {
        "@graphql-eslint/known-type-names": "error"
      }
    }
```

All GraphQL is defined in `src/graph`, e.g.

```
src/
  graph/
    company/
      typeDefs.js
    group/
      typeDefs.js
    ...
```

It finds all the correct files, but it cannot find any types within.

```
$ yarn lint
yarn run v1.22.17
$ eslint ./src

src/graph/company/typeDefs.js
  0:0  error  Parsing error: Unable to find any GraphQL type definitions for the following pointers:
        
          - ./src/**/*.graphql
...
```

Each file has the same error. Here's an example `typeDefs.js` file:

```js
import { gql } from 'apollo-server-express'

export default gql`
  type Company {
    id: ID!
    name: String!
    ...
  }

   ...
}
```

I would greatly appreciate any help or pointers. I assume I'm missing something basic. I've tried adding `parserOptions.schema`, but I don't have a schema to point to, and it complains about `schema.graphql` not existing then (which it doesn't).

Thanks!

**Comment by dimaMachina** [comment, marked as answer]
Hi, could you create a reproduction repo?
